nerdctl/Dockerfile

```
@@ -0,0 +1,17 @@
+FROM quay.io/cybozu/golang:1.17-focal 
+
+ARG NERDCTL_VERSION=v0.20.0
+
+RUN apt-get update \
+        && apt-get install -y git make
+
+ENV SRC_DIR /nerdctl
+WORKDIR $SRC_DIR 
+RUN git clone https://github.com/containerd/nerdctl.git $SRC_DIR \
+        && git checkout refs/tags/$NERDCTL_VERSION
+
+RUN make \
+        &&  make install
```

```suggestion
RUN make && make install
```

It's not long, just one line.

---

Fixed by https://github.com/cybozu/neco-containers/commit/0cbd0eeb9193394dd94468a37926500f42b30aff